docs/releases/1.10.2.txt

```diff
@@ -14,3 +14,6 @@ Bugfixes
 
 * Allowed combining ``contrib.postgres.search.SearchQuery`` with more than one
   ``&`` or ``|`` operators (:ticket:`27143`).
+
+* Fixed issue where pages starting with /de-* are matched to german language.
```

Comment:
Of course this applies to any language that has similar matching, so try to make the note a bit more general.